ci-scripts/JenkinsFile-GitLab

```diff
@@ -354,24 +354,24 @@ pipeline {
               script {
                 def retrieveOAIcovFiles = true
                 try {
-                  myShCmdWithLog('cd lte/gateway && vagrant ssh magma -c "cd magma/lte/gateway && make coverage_oai"', 'archives/magma_vagrant_make_coverage_oai.log', new_host_flag, new_host_user, new_host)
+                  sh('cd lte/gateway && vagrant ssh magma -c "cd magma/lte/gateway && make coverage_oai"')
                 } catch (Exception e) {
                   echo "Let's keep running to have some logs, but not the OAI coverage files"
                   retrieveOAIcovFiles = false
                 }
                 if (retrieveOAIcovFiles) {
                   try {
-                    myShCmd('cd lte/gateway/c/oai && zip -r -qq code_coverage.zip code_coverage/', new_host_flag, new_host_user, new_host)
-                    copyFrom2ndServer('lte/gateway/c/oai/code_coverage.zip', 'archives', new_host_flag, new_host_user, new_host)
+                    sh('cd lte/gateway/c/oai && zip -r -qq ${WORKSPACE}/archives/code_coverage.zip code_coverage/')
+                    //sh('lte/gateway/c/oai/code_coverage.zip', 'archives')
```

Comment:
remove comment

Reply:
This line can be removed. Thanks for catching it